modules/core/src/v2/coins/trx.ts

```diff
@@ -192,4 +205,47 @@ export class Trx extends BaseCoin {
 
     return Buffer.from(sig, 'hex');
   }
+
+  /**
+   * Explain a Tron transaction from txHex
+   * @param params
+   * @param callback
+   */
+  explainTransaction(
+    params: ExplainTransactionOptions,
+    callback?: NodeCallback<TransactionExplanation>
+  ): Bluebird<TransactionExplanation> {
+    return co<TransactionExplanation>(function*() {
+      const txHex = params.txHex || (params.transaction && params.transaction.txHex);
+      if (!txHex || !Tron.safeTronHexString(txHex)) {
+        throw new Error('missing valid txHex string');
+      }
+      if (!params.fee) {
```

Comment:
I think there is a common param validator that we can use to ensure the existence of parameters like this more succinctly